src/goto-programs/goto_convert_class.h

```diff
@@ -672,6 +672,11 @@ class goto_convertt:public messaget
     const symbol_exprt &function,
     const exprt::operandst &arguments,
     goto_programt &dest);
+  void do_havoc_object_slice(
+    const exprt &lhs,
+    const symbol_exprt &function,
+    const exprt::operandst &arguments,
+    goto_programt &dest);
```

Comment:
I'd suggest to add docs, but no other function in this file has Doxygen comments, so maybe we should address this on a separate PR.